### ISSUE TITLE
Forge 1.20.1 Regions Unexplored Missing Recipes

### ISSUE THREAD

**Issue by Zanklev** [MehVahdJukaar/sawmill#45]
On Forge 1.20.1 with the latest version of

Regions Unexplored
Sawmill
Moonlight Lib

Oddly missing numerous recipes for fences and various other items including things like bridges or doors from Macaw's mods
while Regions Unexplored is installed. Removing Regions Unexplored fixes the missing recipes.

**Comment by KirboSoftware** [comment]
:wave: Regions Unexplored maintainer here, 

![image](https://github.com/MehVahdJukaar/sawmill/assets/80041873/ac583064-7f83-41f3-bbbf-87a89be321b4)

Left is without regions unexplored installed
Right is with regions unexplored installed

Somehow the stick and stick depended recipes don't appear when using regions unexplored. I don't know if this is a us issue or if this is on sawmill.

**Comment by MehVahdJukaar** [comment]
Yeah it's somehow not finding a cost for sticks. For all but birch. Does that one in parcirular have anything specific to it? My guess is that while trying to de compose the branch recipe which requires sticks it tries to find a cost for sticks but can't as the cheapest recipe branch that leads to stick is the one for branch itself. In other world that recipe path is a recursive one which is something that it fails to handle, also because it usually happens unfrequently

**Comment by KirboSoftware** [comment]
Is there a way to manually exclude said recipe to avoid this entire issue?

**Comment by MehVahdJukaar** [comment]
i hardcoded the price of sticksto always be fixed. that shuld get rid of it. I'll do some more reasearch onto why this actually happens and how to add a proper fix at a later date. Also yeah the mod has a config to hardcoded prices and one to exclude recipes from mods
